static.html

```diff
@@ -11,7 +11,39 @@
 
 <body>
   <div id="root">
-    <!-- insert your tags here -->
+    <div id ="toolbare">
```

Comment:
You misspelled the id for this div. It should be `toolbar`. That's why it doesn't pickup the expected styling from the `style.css` file.